src/lib.rs

```diff
@@ -388,8 +388,11 @@ mod numeric;
 pub mod types;
 mod util;
 
-use crate::errors::EnvmntError;
+pub use crate::errors::EnvmntError;
+pub use crate::errors::EnvmntResult;
```

Comment:
since result should be in types module and type module is public, i guess we don't need to add the pub modifier here. right?

Reply:
You're right. I kind of fell back to this out of habit, tending to have only one exported surface, all via lib.rs. But that is totally just my thing. It actually does not even play along well with rustdoc. Will fix.

Reply:
Ok, had a look at this. The linter caught the "unreachable_pub" on the type alias. The only way other is to make the errors submodule public.

Your choice. Do you want a flat structure for the library (export at lib.rs level) or a public error submodule?

Given the nature of the lib as a collection of functions and not many types around, I'd go for the former, unless there is an explicit concept to be made out of the error handling. That is a more a question about how the documentation would end up looking like than anything else.

Reply:
err module is public already and i think should stay that way. i was just thinking just removing the pub before the use and putting it inthe mod itself (i think its like that already).

Reply:
Done